From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: ABI fix: add WB_SYNC_HOLD enum writeback_sync_modes

This is only to keep the ABI stable for now, this patch
needs to be removed next chance we get when the ABI changes
as it's not needed at all.

This is here because 2.6.27.13 removes this enum.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/writeback.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/writeback.h
+++ b/include/linux/writeback.h
@@ -30,6 +30,7 @@ static inline int task_is_pdflush(struct
 enum writeback_sync_modes {
 	WB_SYNC_NONE,	/* Don't wait on anything */
 	WB_SYNC_ALL,	/* Wait on every mapping */
+	WB_SYNC_HOLD,	/* DOES NOTHING, DO NOT USE, ONLY HERE FOR ABI issue */
 };
 
 /*
